collection/table-headers.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Table Headers
+
+--- 
+
+The `Table` collection widget supports a header row and/or column
+which can be formatted to stand out from the data
+
+Although these samples don't show it, headers can be more complex objects which could trigger table-specific operations like sorting or filtering.
+
+There is specific support for a "sticky" header, which remains at the top or left of the
+container while the data scrolls.  This support uses additional callbacks, `CreateHeader` and `UpdateHeader`.
+
+```go
+package main
+
+import (
+    "fmt"
+
+    "fyne.io/fyne/v2"
+    "fyne.io/fyne/v2/app"
+    "fyne.io/fyne/v2/widget"
+)
+
+var dataTemplateText = "0123456789012345"
+var sampData = []string{
```

Comment:
It's not a great idea to use 1-dimensional data for a demonstration of a 2-dimensional output.

Reply:
The sample renders different data in each column and the pattern doesn't repeat till 5 rows later.  It doesn't *look* like 1-dimensional data.  
But the point  of the sample is the headers, and maybe it would be simpler and less distracting to put the same placeholder text in all the data cells, or even leave them empty?

Reply:
To clarify, are you concerned about line 25, `var dataTemplateText = "0123456789012345"`?  I thought the concern was about the sample data, at lines 26-31 which is indeed a 1-d vector of strings.  
If it's about line 25, please clarify what you'd rather have there (my idea was you can tell at a glance that this string is 16 runes long).  If it's about the subsequent lines, I could have the sample show blanks in the data cells.  That would focus attention on the headers, which is the point of this sample.